SkatkovAM/lab_2/lab2.cpp

```diff
@@ -0,0 +1,109 @@
+
+#include <iostream>
+
+#include <ctime>
+using namespace std;
+void basicArrayInit(int * arr, int size, int max){
+    for(int i=0; i<size;i++){
+        arr[i] = rand() % max+ 1;
+    }
+}
+void oddArrayInit(const int * basic_arr,int *odd_arr, const int size, int max){
```

Comment:
Instead of filling odd_arr with "impossible" values it's better to create and store variable that represents size of that array which will be <= size